src/coreComponents/common/MpiWrapper.cpp

```diff
@@ -32,6 +32,12 @@
 namespace geos
 {
 
+#ifdef GEOS_USE_MPI
+MPI_Comm MPI_COMM_GEOSX;
```

Comment:
maybe we can change that `MPI_COMM_GEOSX` to `MPI_COMM_GEOS` ?

Reply:
For sure!